azure-pipelines.yml

```diff
@@ -4,159 +4,551 @@ variables:
+                    pod install > /dev/null
+                  workingDirectory: '$(Agent.BuildDirectory)/iOSHost'
+              - task: Bash@3
+                displayName: 'Install XCTestHtmlReport for publishing result'
+                inputs:
+                  targetType: inline
+                  script: |
+                    brew install xctesthtmlreport
+                  workingDirectory: '$(System.DefaultWorkingDirectory)'
+              - task: NodeTool@0
+                inputs:
+                  versionSpec: '14.x'
+                displayName: 'Install Node.js'
+              - task: Cache@2
+                inputs:
+                  key: 'pnpm | "$(Agent.OS)" | $(ClientSdkProjectDirectory)/pnpm-lock.yaml'
+                  path: $(Pipeline.Workspace)/.pnpm-store
+                displayName: Cache pnpm
+              - task: CmdLine@2
+                displayName: 'Setup pnpm'
+                inputs:
+                  script: |
+                    corepack enable
+                    corepack prepare pnpm@7.30.1 --activate
+                    pnpm config set store-dir $(Pipeline.Workspace)/.pnpm-store
+              - task: CmdLine@2
+                displayName: 'Install client sdk dependencies'
+                inputs:
+                  script: |
+                    pnpm install
+                  workingDirectory: '$(ClientSdkProjectDirectory)'
+              - task: CmdLine@2
+                displayName: 'Build client sdk'
+                inputs:
+                  script: |
+                    pnpm build-force-blazor
+                  workingDirectory: '$(ClientSdkProjectDirectory)'
+              - task: Bash@3
+                displayName: 'Run sample test app in background'
+                inputs:
+                  targetType: inline
+                  script: 'nohup pnpm start-test-app &'
+                  workingDirectory: '$(ClientSdkProjectDirectory)'
+              - task: Bash@3
+                displayName: 'Disable Slow Animation'
+                inputs:
+                  targetType: inline
+                  script: cd ~/Library/Preferences defaults write com.apple.iphonesimulator SlowMotionAnimation -bool NO
+                  workingDirectory: '$(System.DefaultWorkingDirectory)'
+              - task: Bash@3
+                displayName: 'iOS UI/E2E Tests'
+                inputs:
+                  targetType: inline
+                  script: "/usr/bin/xcodebuild -configuration Release -workspace /Users/runner/work/1/iOSHost/$(IOSSdkWorkspace).xcworkspace -scheme $(IOSSdkScheme) -sdk iphonesimulator -parallel-testing-enabled YES -parallel-testing-worker-count 2 -destination 'platform=iOS Simulator,name=iPhone 14 Pro,OS=16.2' -screenshot-enabled=\"YES\" -test-iterations 3 -retry-tests-on-failure -quiet -resultBundlePath TestResults test > report.out 2>&1\nif [[ $? == 0 ]]; then echo \"E2E Test passes successfully\"; exit 0; else echo \"E2E Test failed\"; exit 1; fi;"
```

Comment:
could you please double check on test-iteration and retry-tests-on-failure parameters? I believe I have removed them. And also I am pretty sure there is no report.out file.

Please check here: https://github.com/OfficeDev/microsoft-teams-library-js/pull/1995/files

Reply:
Fixed